kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/config/HazelCastSinkConfig.scala

```diff
@@ -18,7 +18,7 @@ package com.datamountaineer.streamreactor.connect.hazelcast.config
 
 import java.util
 
-import com.datamountaineer.streamreactor.temp.{ErrorPolicySettings, KcqlSettings, NumberRetriesSettings, ThreadPoolSettings}
+import com.datamountaineer.streamreactor.temp._
```

Comment:
It's generally a bad practice for 'wildcard' imports. Are you using more then those 4 imports?